corehq/apps/style/static/style/js/main.js

```diff
@@ -1,287 +1,286 @@
-var COMMCAREHQ = {};
-
-COMMCAREHQ.icons = {
-    GRIP:   'icon-resize-vertical icon-blue fa fa-arrows-v',
-    ADD:    'icon-plus icon-blue fa fa-plus',
-    COPY:   'icon-copy icon-blue fa fa-copy',
-    DELETE: 'icon-remove icon-blue fa fa-remove',
-    PAPERCLIP: 'icon-paper-clip fa fa-paperclip'
```

Comment:
Would it be easy to turn these into classes with nice names like `hq-grip`? It's hard to remember all those classes, and the symbolic names were nice.